fuzz/fuzz_pom_xml.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from cve_bin_tool.log import LOGGER
+from cve_bin_tool.cvedb import CVEDB
+import generated.pom_xml_pb2 as pom_xml_pb2
+import sys
+import tempfile
+from pathlib import Path
+
+import atheris
+import atheris_libprotobuf_mutator
+from google.protobuf.json_format import MessageToDict
```

Comment:
```suggestion
import sys
import tempfile
from pathlib import Path

import atheris
import atheris_libprotobuf_mutator
import generated.pom_xml_pb2 as pom_xml_pb2
from google.protobuf.json_format import MessageToDict

from cve_bin_tool.cvedb import CVEDB
from cve_bin_tool.log import LOGGER
```

isort just wants the "local" includes at the bottom instead of the top, so this should help.  If you can't use the pre-commit hooks for any reason, you can always run `isort <filename>` to get it to fix things instead.